cornice_swagger/swagger.py

```diff
@@ -148,6 +148,12 @@ def from_path(self, path):
         param_names = [comp[1:-1] for comp in path_components
                        if comp.startswith('{') and comp.endswith('}')]
 
+        # handle traverse and subpath
+        # docs.pylonsproject.org/projects/pyramid/en/latest/narr/hybrid.html
+        for subpath_marker in ('*subpath', '*traverse'):
```

Comment:
I think we can do this here if we assume that subpaths only occur outside `{ }`.